docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     image: wordpress
     ports:
       - 8888:80
+      - 9999:443
```

Comment:
Is the change of port required here, why not keep using `8888` and use `8888:443`?

Reply:
I don't think it's required, I just did it so you'd know which version of the site you were working against during development. It's not usual in my experience to expect the protocol to change without the port changing. No reason it can't change though.